modules/mcuboot/CMakeLists.txt

```diff
@@ -159,6 +164,8 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
   endif()
 
   if(CONFIG_SIGN_IMAGES)
+    set(app_core_binary_name app_update.bin)
```

Comment:
Should `app_update.bin` be configurable or maybe `${CONFIG_KERNEL_BIN_NAME}_update.bin` ?
So that it follows the app name, if user changes from `zephyr.elf` to `awesome_app.elf`, then the update image follow suit and becomes `awesome_app_update.bin`.

Reply:
I agree that this is better, but I'm afraid of breaking compatibility with tools that depend on the name of the generated update bin

Reply:
could it be investigated whether this is the case.
Doesn't the zip contain a meta file that describes exactly which file in the zip that goes to which core, so the tools are using that ?

Reply:
To me it looks as if the file name is being passed on from CMake and added to a manifest here:
https://github.com/nrfconnect/sdk-nrf/blob/4af2c283eea8e2a1d5f7663a0ebe659e2cbd7d82/cmake/fw_zip.cmake#L27
https://github.com/nrfconnect/sdk-nrf/blob/4af2c283eea8e2a1d5f7663a0ebe659e2cbd7d82/scripts/bootloader/generate_zip.py#L63

Disclaimer, there could be details in this i'm not familiar with.

Reply:
I will double check, if its the case then I will change it.